SPECS/intel-igc.spec

```
@@ -44,6 +44,7 @@ Requires(postun): /usr/sbin/depmod
 
 # This RPM obsoletes XCP-ng specific RPM igc-module
 Obsoletes: igc-module < 5.10.200-2
+Provides: igc-module = 5.10.200-2
```

As I understand the guidelines, and what seems logical to me, is to indicate the current version and release here.

```suggestion
Provides: igc-module = %{version}-%{release}
```

---

indeed!